website/src/auth.tsx

```diff
@@ -268,6 +269,11 @@ export const UserProvider = (props: { children: any }) => {
     })
   }
 
+  function signOutUser() {
+    setUser(null)
+    user?.signOut()
```

Comment:
should we do the `setUser(null)` in a callback? In case signing out can error or something (ie. bad network)